src/pages/billing/List.js

```diff
@@ -233,9 +233,7 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
     Utils.withBusyState(setIsLoadingProjects)
   )(async () => setBillingProjects(_.sortBy('projectName', await Ajax(signal).Billing.listProjects())))
 
-  const loadAlphaSpendReportMember = _.flow(
-    reportErrorAndRethrow('Error loading user group membership')
-  )(async () => {
+  const loadAlphaSpendReportMember = reportErrorAndRethrow('Error loading user group membership')(async () => {
```

Comment:
This is cleanup from a previous PR-- I didn't notice the lint warning about calling `flow` with a single argument.